src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        ColorSupplier supplier = new ColorSupplier();
```

Comment:
lets make ColorSupplier a global variable, so it is going to be created just once instead of each time this method is invoked